role-management-commands/role.md

```diff
@@ -1,6 +1,6 @@
 # role
 
-<TODO: describe>
+This command allows you to assign or remove a role from a user.
```

Comment:
Clarify it works on a vice-versa basis e.g user does not have role => adds it, user has role => removes it

Reply:
Also important thing I forgot: with this command you can target a user not in the server, and doing so will create a persistence for when they join... same exact command will remove it
![paveikslas](https://github.com/MoonlightCapital/MoonlightBot-docs/assets/19293406/2e3345a0-196c-493f-acab-dae04da2bf6c)